README.md

```
@@ -2,313 +2,1389 @@
+- [Ashish Mehra](https://ashishmehra.dev)
+- [Ashish Namdeo](https://ashishnamdeo.com)
+- [Ashlok Chaudhary](https://ashlok.dev) [Full Stack Developer || Devops]
+- [Ashwin Hariharan](https://ashwinhariharan.com/software-engineering) [Full Stack Developer]
+- [Aster Li](https://asterjuneli.com)
+- [Atanas Atanasov](https://atanas.info)
+- [Atharva Kote](https://atharvakote.netlify.app) [Full Stack Developer | Deveops]
+- [Atul Kumar Awasthi](https://atultheportfolio.netlify.app)
+- [Auroob Ahmad](https://auroob.github.io/dev-port)
+- [Austin Gericke](https://www.austingericke.com)
+- [Austin Pham](https://auspham.dev)
+- [Austin Serb](https://www.austinserb.com)[Full Stack Developer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://www.austinserb.com)